classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'class_student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @student = []
```

Comment:
- [ ] In Student class I think it would be great if you add a method `\that serves as the 'belongs-to' side (a student belongs to a classroom). Here's a [Link to the article with examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) so kindly add it in student class def classroom 🙏🏻✨🔥💯